vendor.yml

```diff
@@ -1,68 +1,35 @@
 ---
 js:
-  regular:
-  - node_modules/jquery/dist/jquery.js
-  - node_modules/bootstrap/dist/js/bootstrap.js
-  - node_modules/bootbox/bootbox.js
-  - node_modules/filesize/lib/filesize.js
-  - node_modules/lodash/lodash.js
-  - node_modules/moment/moment.js
-  - node_modules/chart.js/dist/Chart.js
-  - node_modules/splitargs/src/splitargs.js
-  - node_modules/toastr/toastr.js
-  - node_modules/xterm/dist/xterm.js
-  - node_modules/xterm/dist/addons/fit/fit.js
-  - node_modules/js-yaml/dist/js-yaml.js
-  - node_modules/codemirror/lib/codemirror.js
-  - node_modules/codemirror/mode/yaml/yaml.js
-  - node_modules/codemirror/addon/lint/lint.js
-  - node_modules/codemirror/addon/lint/yaml-lint.js
-  minified:
-  - node_modules/jquery/dist/jquery.min.js
-  - node_modules/bootstrap/dist/js/bootstrap.min.js
-  - node_modules/bootbox/bootbox.js
-  - node_modules/filesize/lib/filesize.js
-  - node_modules/lodash/lodash.min.js
-  - node_modules/moment/min/moment.min.js
-  - node_modules/chart.js/dist/Chart.min.js
-  - node_modules/splitargs/src/splitargs.js
-  - node_modules/toastr/build/toastr.min.js
-  - node_modules/xterm/dist/xterm.js
-  - node_modules/xterm/dist/addons/fit/fit.js
-  - node_modules/js-yaml/dist/js-yaml.min.js
-  - node_modules/codemirror/lib/codemirror.js
-  - node_modules/codemirror/mode/yaml/yaml.js
-  - node_modules/codemirror/addon/lint/lint.js
-  - node_modules/codemirror/addon/lint/yaml-lint.js
+  - 'jquery/dist/jquery.js'
```

Comment:
Are the quotes needed around each dependency? I have forgot to add them for the Angular section during the merge.

Reply:
Maybe not. But I'm concerned with prefixing 'node_modules' to angular entries. This PR adds it programatically (https://github.com/1138-4EB/portainer/blob/15fb8b60d83c8108f277cb5a87a3fca6947397de/gruntfile.js#L66), so I think there is no need to write it in vendor.yml. Did you need it?

Reply:
Yeah sorry about that, I rushed a bit the conflict resolution.